zokrates_field/src/bn128.rs

```diff
@@ -4,372 +4,12 @@
 // @author Jacob Eberhardt <jacob.eberhardt@tu-berlin.de>
```

Comment:
author tags should move to the `lib.rs` file and not stay in this bn128 stub

Reply:
moved